Code/GraphMol/Descriptors/catch_tests.cpp

```diff
@@ -30,18 +30,9 @@ TEST_CASE("Kier kappa2", "[2D]") {
         // Table 5 from the paper
         {"c1ccccc15.Cl5", 1.987},        {"c1ccccc15.F5", 1.735},
         {"c1ccccc15.[N+]5(=O)O", 2.259}, {"c1ccccc15.C5(=O)C", 2.444},
-#if 0
-      // expected values from paper (differences are due to hybridization mismatches)
-        {"c1ccccc15.N5(C)C", 2.646},
-        {"c1ccccc15.C5(=O)N", 2.416},
-        {"c1ccccc15.C5(=O)O", 2.416},
-        {"c1ccccc15.S5(=O)(=O)C", 2.617},
-        {"c1ccccc15.O5", 1.756},
-#else
```

Comment:
These are documentation and should remain in some form.
Perhaps we could include something like:
```
        /* expected values from paper (differences are due to hybridization
          mismatches)
          {"c1ccccc15.N5(C)C", 2.646},
          {"c1ccccc15.C5(=O)N", 2.416},
          {"c1ccccc15.C5(=O)O", 2.416},
          {"c1ccccc15.S5(=O)(=O)C", 2.617},
          {"c1ccccc15.O5", 1.756},
        */
```